models/tmp/stg_amazon_ads__ad_group_history_tmp.sql

```diff
@@ -1,2 +1,4 @@
+{{ config(enabled=var('ad_reporting__amazon_ads_enabled', True)) }}
```

Comment:
You need to add the disable config to the `tmp` model as well in order for it to be properly disabled. Be sure to add this config to each tmp model

Reply:
done